package.json

```diff
@@ -41,10 +41,8 @@
     "chai-things": "^0.2.0",
     "coveralls": "^2.11.12",
     "eslint": "^2.9.0",
-    "eslint-config-airbnb": "^9.0.1",
+    "eslint-config-airbnb-base": "^3.0.0",
```

Comment:
This is the same (outdated) version you used from the normal airbnb preset, but without all the react stuff